src/commands/createFunction/openAPISteps/OpenAPICreateStep.ts

```diff
@@ -109,3 +132,33 @@ async function addAutorestSpecificTypescriptDependencies(context: IFunctionWizar
         return data;
     });
 }
+
+async function addAutorestSpecificPythonDependencies(context: IFunctionWizardContext): Promise<void> {
+
+    let oldRequirements: string = '';
+    let newRequirements: string = '';
+    const requirementsPath: string = path.join(context.projectPath, requirementsFileName);
+
+    // Current list of dependencies to be added to support stencil generated projects.
+    const dependenciesToAddArray: string[] = ['msrest'];
+
+    // Some custom requirements might not have newline character added already.
+    const dependenciesToAdd: string = `\n${dependenciesToAddArray.join('\n')}\n`;
+
+    if (await fse.pathExists(requirementsPath)) {
+        try {
+            oldRequirements = (await fse.readFile(requirementsPath)).toString();
+            newRequirements = oldRequirements.concat(dependenciesToAdd);
```

Comment:
This should check if `oldRequirements` already has 'msrest' before trying to add it

Reply:
Done. Made it better for future usage as well, to make I also take care of a deleted requirements.txt - making this self-sufficient.